### PR TITLE
Add language variants knob to Caption

### PR DESCRIPTION
**Pull request by alex-magana**
Relates to #331, #370

Depends on #378  

**Overall change:** Adds the support for examining Psammead components' behaviour
in the different language services using the storybook select knob

**Code changes:**
- Adds stories to the `psammead-caption` component that use the select knob.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by alex-magana** [comment]
The `continuous-integration/jenkins/branch ` fails with
`npm ERR! Missing: @bbc/psammead-storybook-helpers@^0.1.0`
This should be resolved once #378 is merged and published.

File: CHANGELOG.md
Comment (dr3) [inline CHANGELOG.md]: We dont need to bump/changelog for a change that doesnt reach outside of the `packages` dir :)

**Review by dr3 — DISMISSED** [review]
LGTM after removing the changelog and installing the published package. Great work, it looks awesome

**Review by dr3 — COMMENTED** [review]
LGTM, will need to wait for the other PR to be merged though

**Comment by pjlee11** [comment]
Blocked on #378

**Review by amywalkerdev — DISMISSED** [review]
👍

File: packages/components/psammead-caption/package.json
Comment (dr3) [inline packages/components/psammead-caption/package.json]: On second thoughts, this should be `0.3.4` as it doesn't affect the primary code

**Review by dr3 — CHANGES_REQUESTED** [review]

File: packages/components/psammead-caption/package.json
Comment (andrew-nowak) [reply packages/components/psammead-caption/package.json]: Done

**Review by dr3 — DISMISSED** [review]

**Review by amywalkerdev — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM. Happy for this to be merged.

**Pull request merged**
